python/tests/test_highlevel.py

```diff
@@ -2445,6 +2445,103 @@ def test_tree_node_edges(self):
                         assert edge.right >= tree.interval.right
             assert np.all(edge_visited)
 
+    def verify_individual_vectors(self, ts):
+        verify_times = np.repeat(np.nan, ts.num_individuals)
+        verify_populations = np.repeat(tskit.NULL, ts.num_individuals)
+        for ind in ts.individuals():
+            if len(ind.nodes) > 0:
+                t = {ts.node(n).time for n in ind.nodes}
+                p = {ts.node(n).population for n in ind.nodes}
+                assert len(t) <= 1
+                assert len(p) <= 1
+                verify_times[ind.id] = t.pop()
+                verify_populations[ind.id] = p.pop()
+
+        times = ts.individual_times
+        populations = ts.individual_populations
+        assert np.array_equal(times, verify_times, equal_nan=True)
```

Comment:
This makes me wonder if individuals with no nodes should have an infinite time rather than nan? np.inf == np.inf is True.

Reply:
Yeah, fair enough.